src/components/Matchup/Matchup.tsx

```diff
@@ -357,36 +357,110 @@ function Matchup(props: MatchupProps) {
       
       return swappedPlayers;
    }
-   
-   return (<div data-testid="Matchup" className="border p-1 border-slate-50 rounded-lg">
-      <h1>{props.league}</h1>
-      <div className='team-container'>  
-       
-         {homeFilteredPlayers && awayFilteredPlayers && homeFilteredPlayers.map((p, index) => (          
-         <Team key={index} 
-         homePlayerName={p.playerName} homePlayerPosition={p.playerPosition} homePlayerPoints={p.playerPoints} 
-         homePlayerSubPoints={p.subPoints} homePlayerCaptainPoints={p.captainPoints} homePlayerSubbedInFor={p.subbedInFor} homePlayerSubbedOutFor={p.subbedOutFor} 
-
-         awayPlayerName={awayFilteredPlayers[index].playerName} awayPlayerPosition={awayFilteredPlayers[index].playerPosition} awayPlayerPoints={awayFilteredPlayers[index].playerPoints} 
-         awayPlayerSubPoints={awayFilteredPlayers[index].subPoints} awayPlayerCaptainPoints={awayFilteredPlayers[index].captainPoints} awayPlayerSubbedInFor={awayFilteredPlayers[index].subbedInFor} awayPlayerSubbedOutFor={awayFilteredPlayers[index].subbedOutFor}></Team>))}
-         {/* {props.matchupData && props.matchupData.homePlayers && props.matchupData.homePlayers.map((p, index) => ( <Team key={index} homePlayer={p} awayPlayer={props.matchupData.awayPlayers[index]}></Team>))} */}
-         
+
+   return (
+      <div data-testid="matchup" className="border p-1 border-slate-50 rounded-lg">
+         <h1>{props.league}</h1>
+         {
+            homeFilteredPlayers &&
+            awayFilteredPlayers &&
+            homeFilteredPlayers.sort(
+               (n1: PlayerData, n2: PlayerData) => n1.subOrder < n2.subOrder ? -1 : 1).map((p, index) => (
+                   <Team
+                     key={index}
+                     homePlayerName={p.playerName}
+                     homePlayerPosition={p.playerPosition}
```

Comment:
Broke these out vertically so can easily read which prop was being used, and how.

Reply:
nice!